### DISCUSSION TITLE
Reference your GitHub code in a Markdown file

### DISCUSSION THREAD

**Discussion by AlexHedley** [community/community#30026] [Mobile]
Will the mobile app be updated to display this feature?

https://github.com/stevemar/code-reference-in-readme

https://www.stevemar.net/github-code-in-readme/

![image](https://user-images.githubusercontent.com/1573469/185557722-5ca9123d-c9bf-48fd-a26e-758ed4c058be.png)

**Comment by taki-on** [comment]
Thank you for your feedback. The current markdown renderer supports basic markdown rendering. GitHub.com supports more features like code rendering, LaTeX supports, etc. We will discuss how to support them on Mobile. Thank you for your feedback!
